server/db/populate-db.sql

```diff
@@ -13,6 +14,78 @@ INSERT INTO languages (code, name) VALUES ('be','Bengali');
 INSERT INTO languages (code, name) VALUES ('fr','French');
 
 
-INSERT INTO documents (name, from_language_code, to_language_code, status, submission_date, due_date, owner_id, format, content) values ('Birth Certificate', 'ar','en', 'Waiting', '2019-01-20','2019-02-20','db5b2e53-acd2-4af0-bd91-8554d9dc09ef','Text', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam');
-INSERT INTO documents (name, from_language_code, to_language_code, status, submission_date, due_date, owner_id, format, content) values ('University Transcript','zh-cn','en', 'Processing', '2019-07-20','2019-08-02', 'a2deeae9-1d8c-44ef-a187-a85dfa56fee8', 'Text', 'cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.');
+INSERT INTO documents (name, from_language_code, to_language_code, status, submission_date, due_date, owner_id, format, content) values ('Birth Certificate', 'ar','en', 'Waiting', '2019-10-08',NOW() - INTERVAL '2 DAY','a2deeae9-1d8c-44ef-a187-a85dfa56fee8','Text', E'كان لايزال " لويس باستور" مدرسا للكيمياء في جامعة ستراسبورغ حين التقى بـ"ماري آن لوران" ابنة رئيس الجامعة التي غيرت حياته وليصبح زوجها  عالما أنقذ ملايين البشر وترك بصمة وأثراً كبيراً لدى البشرية.
```

Comment:
Can you also change the submission_date to a relative date. We can have for example always 3 documents which are new (so using todays' date with NOW()) and the rest older than today so they dont have the new label.